Dockerfile

```
@@ -0,0 +1,94 @@
+ARG UBUNTU_VERSION=18.04
+FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu${UBUNTU_VERSION}
+# python3.6 is provided by ubuntu18.04
+# python3.5 is provided by ubuntu16.04
+
+MAINTAINER blueoil-admin@leapmind.io
```

Change to use `LABEL`
```suggestion
LABEL maintainer "LeapMind Inc. <blueoil-admin@leapmind.io>"
```